resources/views/pages-legacy/leaderboardinfo.blade.php

```diff
@@ -112,19 +116,22 @@
 
             echo "<li>Manage Entries</li>";
             echo "<div>";
-            if (!empty($lbData['Entries'])) {
+            if (!$orderedEntries->isEmpty()) {
                 echo "<tr><td>";
                 echo "<form method='post' action='/request/leaderboard/remove-entry.php' onsubmit='return confirm(\"Are you sure you want to permanently delete this leaderboard entry?\")'>";
                 echo csrf_field();
                 echo "<input type='hidden' name='leaderboard' value='$lbID' />";
                 echo "Remove Entry:";
                 echo "<select name='user'>";
                 echo "<option selected>-</option>";
-                foreach ($lbData['Entries'] as $nextLBEntry) {
+                foreach ($orderedEntries as $nextLBEntry) {
                     // Display all entries for devs, display only own entry for jr. devs
-                    if (($user == $nextLBEntry['User'] && $permissions == Permissions::JuniorDeveloper) || $permissions >= Permissions::Developer) {
-                        $nextUser = $nextLBEntry['User'];
-                        $nextScore = $nextLBEntry['Score'];
+                    if (
+                        ($user === $nextLBEntry->user->User && $permissions === Permissions::JuniorDeveloper)
```

Comment:
As things are migrated it may be better to implement modern roles at the same time.

Reply:
Agreed, done in latest.